platform-sdk/swirlds-merkle/src/main/java/com/swirlds/merkle/map/MerkleMapMetrics.java

```diff
@@ -74,7 +72,7 @@ public static boolean isRegistered() {
     }
 
     /**
-     * Registers the {@link MerkleMap} statistics with the specified {@link Platform} instance.
+     * Registers the {@link MerkleMap} statistics with the specified Platform instance.
```

Comment:
The `@link` is missing

Reply:
This link can no longer exist without adding a circular dependency on swirlds common and swirlds core.